editor.planx.uk/src/routes/views/standalone.tsx

```diff
@@ -24,14 +24,14 @@ const standaloneView = async (req: NaviRequest) => {
   const teamSlug =
     req.params.team || (await getTeamFromDomain(window.location.hostname));
   const data = await fetchDataForStandaloneView(flowSlug, teamSlug);
-
+  console.log(data);
   const {
     flows: [{ team, settings: flowSettings }],
     globalSettings,
   } = data;
 
   const state = useStore.getState();
-  state.setFlowNameFromSlug(flowSlug);
+  state.setFlowName(flowSlug);
```

Comment:
similar to above, can we clarify/confirm we're actually still expecting to pass `flowSlug` as the argument to `setFlowName()` now that we're _starting from_ name and generating slug?

Reply:
I was just trying to get a place to test this workflow, where would I be able to see and interact with the Standalone view?

Reply:
Right standalone views are a little tricky to get to - the best example is on the "Invite to Pay" journey.

Try these steps locally: 
- Set up a simple local test flow in a council that has GOV.UK Pay configured (eg Lambeth)
- This flow should have these components at minimum: FindProperty, Checklist that sets data field "proposal.projectType" with at least one option, SetValue with "fee = 1", Pay referencing "fee" data field and "Allow users to invite someone else to pay" option toggled on, Send (select 'email' option even though it won't be configured locally & will fail), Confirmation - 
- Publish your flow
- From the `/published` link of the public form, go trhough the test service and when you reach Pay, click "Invite to pay" then fill out the form using your own contact info so you get the invitation email
- Check your inbox, there should be an invitation to pay with a "magic link" - open it 
- That's a "standalone" page ! "Standalone" because it's a disconnected journey from the main service

There's a lot of new stuff in those steps, so please don't hesitate to shout if you get stuck at any point & want to screenshare!

Reply:
Ah! Okay cool, I'll have a look.

Can I ask why this standalone page is ``state.SetFlowName()`` should it not just read the flowName from the global store instead of setting it?

Reply:
So because this is a short separate journey only accessed by someone invited to _pay_ for the application (different than the person who has filled out the service), the original _flow_ itself never actually gets initiated or created when this "standalone" route loads, therefore flow attributes won't be set in global store! It's just a discrete, short set of static questions not controlled via the Editor, so we repeat a couple of these small details like setting flow & team names on the fly.